medmodels/medrecord/medrecord.py

```diff
@@ -1232,3 +1233,168 @@ def __getitem__(
             return self.select_nodes(key)
 
         return self.select_edges(key)
+
+    def _describe_group_nodes(self) -> pl.DataFrame:
+        """Creates a summary of group nodes and their attributes.
+
+        Returns:
+            pl.DataFrame: Dataframe with all nodes in medrecord groups and their attributes.
+        """
+        df_schema = {
+            "Nodes Group": pl.String,
+            "Count": pl.Int32,
+            "Attribute": pl.String,
+            "Info": pl.String,
+        }
+
+        node_groups = [pl.DataFrame(schema=df_schema)]
+
+        groups = set(self.groups)
+
+        for group in sorted(groups):
+            nodes = self.group(group)["nodes"]
+
+            schema = (
+                self.schema.group(group).nodes if group in self.schema.groups else None
+            )
+
+            node_info = extract_attribute_summary(self.node[nodes], schema=schema)
+
+            node_info = node_info.select(
+                [
+                    pl.lit(group).alias("Nodes Group"),
+                    pl.lit(len(nodes)).alias("Count"),
+                    pl.all(),
+                ]
+            )
+
+            node_groups.append(node_info)
+
+        node_table = pl.concat(node_groups)
+
+        if node_table.is_empty():
+            node_table = pl.DataFrame({col: "-" for col in node_table.columns})
+
+        return node_table
+
+    def _describe_group_edges(self) -> pl.DataFrame:
+        """Creates a summary of edges connecting group nodes and the edge attributes.
+
+        Returns:
+            pl.DataFrame: DataFrame with an overview of edges connecting group nodes.
+        """
+        edge_groups = [
+            pl.DataFrame(
+                schema={
+                    "Edges Groups": pl.String,
+                    "Count": pl.Int32,
+                    "Attribute": pl.String,
+                    "Info": pl.String,
+                }
+            )
+        ]
+
+        groups = sorted(set(self.groups))
+
+        for source_group in groups:
+            # edges in groups
```

Comment:
inconsistent comment (the other function doesnt have that comment). probably not needed